spec/unit/util/execution_spec.rb

```diff
@@ -144,6 +155,20 @@ def stub_process_wait(exitstatus)
         call_exec_windows('test command', {}, @stdin, @stdout, @stderr)
       end
 
+      context 'cwd option' do
+        let(:cwd) { 'cwd' }
+        it "should execute the command in the specified working directory" do
```

Comment:
So, this scenario may be one where we might need to look at testing implementation. I can't think of a good way to unit or acceptance test that the process is executing in a different directory other than testing the implementation itself here. @ekinanp if you can think of a way for us to test that scenario outside of implementation testing that would be great, but if not we can just leave this as-is.

Reply:
Going to leave it as-is, the test is easy enough and tests the right thing